src/mixins/text/index.js

```diff
@@ -0,0 +1 @@
+export const addPluralizationIf = (condition) => condition ? 's' : ''
```

Comment:
Is there any library that does this automatically? I know of this one, but I don't know if it works for spanish: ://github.com/blakeembrey/pluralize

Reply:
I can give it a try. There is a pluralize-es but it has not been updated for 4 years now....

Reply:
FYI I've been using this library https://www.npmjs.com/package/pluralize and it's been good so far.

Reply:
It worked for 'respuesta', I suspect there could be some corner cases but all good for now.